test/test_data_collector.py

```diff
@@ -44,21 +45,21 @@ def test_data_collection():
 
     to_test_data = test_collector._data
 
-    assert to_test_data[0].name == 'test_stat_1'
-    assert to_test_data[1].name == 'test_stat_2'
-    assert to_test_data[0].value == 3
-    assert to_test_data[0].unit == 'tests'
-    assert abs(to_test_data[0].timestamp - 130.452) < 0.1
-    assert to_test_data[0].complete
+    assert to_test_data[0].MetricName == 'test_stat_1'
```

Comment:
If we maintain the `Datum` format and only update the serialization, then this test would not have to change